### DISCUSSION TITLE
Has anyone managed to add new preset to the driving profile screen?

### DISCUSSION THREAD

**Discussion by downtime358** [jilleb/MQB-FPA#22] [Exploration and research]
It is possible to modify the existing items on the screen to be something else than what they are from factory.

Like normal is replaced with sport, eco with comfort etc.

But has anyone actually managed to add a new preset to the driving profiles? All who retrofit DCC for example will not have comfort option there. Is there any way possible to add this mode or any other mode there as an additional mode?

**Comment by jilleb** [comment]
Yes, it's possible to add additional settings to a profile, but it's not possible to add an additional profile. For profiles you need additional FECs

**Reply by tru1** [reply]
Hi,

Pls could you explain how to enable/activate Comfort profile? It is not activated by factory in my car, but I installed DCC and want to have Comfort Mode enabled

**Reply by jilleb** [reply]
You have to replace an existing profile. You cannot add the profile to the existing list if you don't have the FEC.

**Comment by downtime358** [comment]
Just replace one of the existing modes with comfort.
I replaced Eco mode.

**Reply by tru1** [reply]
It seems to be easiest way :) But i still want to jeep Eco. 

Do you know how to enable Comfort Mode?

**Comment by downtime358** [comment]
It’s a swap function so you’ll need someone who can generate and inject the proper FeC to the gateway. With this tool it is not possible to do.
